extensions/barrage/src/main/java/io/deephaven/extensions/barrage/chunk/ChunkInputStreamGenerator.java

```diff
@@ -195,182 +195,21 @@ static WritableChunk<Values> extractChunkFromInputStream(
             final DataInput is,
             final WritableChunk<Values> outChunk, final int offset, final int totalRows) throws IOException {
         return extractChunkFromInputStream(options, 1, chunkType, type, componentType, fieldNodeIter, bufferInfoIter,
-                is,
-                outChunk, offset, totalRows);
+                is, outChunk, offset, totalRows);
     }
 
-    static WritableChunk<Values> extractChunkFromInputStream(
+    @Deprecated
```

Comment:
These methods should be removed, probably inlined into the only remaining code that uses them (flight round trip test)

Reply:
Tests are failing due to this method being incomplete - removing it with something specific for the test will resolve them.